### ISSUE TITLE
New Tutorial: convert JS IPFS examples to tutorials

### ISSUE THREAD

**Issue by alanshaw** [ProtoSchool/protoschool.github.io#136]
From @daviddias in https://github.com/ipfs/js-ipfs/issues/1816:

> Perhaps they won't all be doable on ProtoSchool, but going through the list I would say that at least we can:
> 
> * https://github.com/ipfs/js-ipfs/tree/master/examples/browser-mfs
> * https://github.com/ipfs/js-ipfs/tree/master/examples/browser-add-readable-stream
> * https://github.com/ipfs/js-ipfs/tree/master/examples/browser-readablestream
> * https://github.com/ipfs/js-ipfs/tree/master/examples/browser-video-streaming
> * https://github.com/ipfs/js-ipfs/tree/master/examples/custom-libp2p
> * https://github.com/ipfs/js-ipfs/tree/master/examples/explore-ethereum-blockchain
> * https://github.com/ipfs/js-ipfs/tree/master/examples/ipfs-101
> * https://github.com/ipfs/js-ipfs/tree/master/examples/traverse-ipld-graphs

**Comment by terichadbourne** [comment]
Additional notes from @daviddias in the previous issue: 

> These could actually be tutorials to get help from volunteers that are looking for ideas of tutorials to write
> 
> Also worth checking the ones in js-ipfs-api
> https://github.com/ipfs/js-ipfs-http-client/tree/master/examples

**Comment by terichadbourne** [comment]
It looks like https://github.com/ipfs/js-ipfs/tree/master/examples/ipfs-101 would translate nicely into one of our first tutorials on the file API. See issue #91 for more on that proposal.

**Comment by terichadbourne** [comment]
@johnnymatthews Are these all among the examples that you're planning to move into the docs site?

**Comment by johnnymatthews** [comment]
Yeah they are, but that whole _move everything into docs.ipfs.io_ project got superseded by the _help Filecoin exist_ project. So moving stuff over is on the back-burner for now.
